include/mbedtls/config.h

```diff
@@ -294,6 +294,8 @@
 //#define MBEDTLS_AES_SETKEY_DEC_ALT
 //#define MBEDTLS_AES_ENCRYPT_ALT
 //#define MBEDTLS_AES_DECRYPT_ALT
+//#define MBEDTLS_ECDH_GEN_PUBLIC_ALT
+//#define MBEDTLS_ECDH_GEN_PUBLIC_ALT
```

Comment:
I presume you wanted to write `MBEDTLS_ECDH_COMPUTE_SHARED_ALT` here, could you please fix this?

Reply:
Sure, it was a copy\paste error I missed..